config/deploy/prod.rb

```diff
@@ -0,0 +1,6 @@
+ENV['APP_ENV']  ||= 'production'
+require_relative 'server_settings'
+
+server 'sul-sdr-services.stanford.edu', user: 'sdr2service', roles: %w{app}
+
+Capistrano::OneTimeKey.generate_one_time_key!
```

Comment:
It would help if this PR simply copied over all the files from the sdr-config repo and then made the changes to those files in a second commit of this PR.  It might be easier to then see what the second commit changes.  I'm working around this problem using my local workspace for now, so I can review this PR properly.